### PR TITLE
Stations list

### PR DESCRIPTION
**Pull request by gaetan-cordonnier**
Component StationList faor display staions open/closes and with TPE or any

### CODE REVIEW

**Review by ThomasDruart — DISMISSED** [review]
Ok, good job !

File: src/App.jsx [hunk 1]
```diff
@@ -3,16 +3,17 @@ import Footer from "./components/Footer";
...
-    </div>
-  );
+	return (
+		<div>
+			<Reset />
+			<Header />
+			<StationsList />
+			<Footer />
+		</div>
+	);
```
Comment (AlexisCatrix) [inline src/App.jsx:16]: MapLille component disappeared, you forgot to use React router for keep initials components and make a redirection of < StationsList /> into < MapLille /> trough index.jsx in my opinion.

**Review by nordinemadoui — DISMISSED** [review]

**Review by nordinemadoui — DISMISSED** [review]
ok good

**Review by LoicBrassart — DISMISSED** [review]
SnE
On the other hand, you seem to have a problem of indentation step, please fix it (it forces update on a lot of useless lines, making PRs longer to read)

**Review by AlexisCatrix — DISMISSED** [review]
ok

**Review by ThomasDruart — DISMISSED** [review]
ok

**Review by AlexisCatrix — APPROVED** [review]
ok

**Review by ThomasDruart — APPROVED** [review]
ok

**Pull request merged**